capabilities/README.md

```diff
@@ -10,6 +10,7 @@ This table lists all capabilities and links to their descriptions.
 
 | Capability | Owner |
 |------------|-------|
+| [Serverless Runtime](runtime.md) | [@joek](https://github.com/joek) |
```

Comment:
The capabilities should display in alphabetical order. Please place the Serverless Runtime between Logging / Tracing / Monitoring and Service Management.